data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -51,21 +51,21 @@
     private final AuditMessageService auditMessageService;
 
     /**
-     * Constructor for the DataController
+     * Constructor for the DataController.
      *
-     * @param auditableDataService service for providing auditable data for the request
-     * @param auditMessageService  service for sending audit messages
+     * @param auditableDataService service for providing data retrieval service for the request
```

Comment:
I think this needs to be re-worded. A "service for providing a retrieval service" doesn't read properly.
Maybe something like "service for retrieving the data of the request" would be better.

Reply:
Looks like two words were not deleted.  Now it reads "service for retrieving data for the request".